slither/utils/command_line.py

```diff
@@ -332,3 +333,29 @@ def output_printers_json(printer_classes):
         table.append({"index": idx, "check": argument, "title": help_info})
         idx = idx + 1
     return table
+
+
+def check_and_sanitize_markdown_root(markdown_root: str) -> str:
+    # Regex to check whether the markdown_root is a GitHub URL
+    match = re.search(
+        r"(https://)github.com/([a-zA-z-]+)([:/][A-Za-z0-9_.-]+[:/]?)([A-Za-z0-9_.-]*)(.*)",
```

Comment:
Lgtm is complaining `This regular expression includes duplicate character 'z' in a set of characters`,  looks like it is right and there is `z` that can be removed from the regex right?

Reply:
Hmm right, there seems to be a typo `A-Z` resulting in incorrect pattern matching of org/profile name of the URL. 
Fixed it.